PSKoans/Koans/Modules/dbatools/AboutBackupDatabases.ps1

```diff
@@ -1,6 +1,5 @@
-#Requires -Modules dbatools
```

Comment:
The Requires statement should probably stay in, though. 🙂 

`#Requires` is for PowerShell to know whether the script is expected to work (it wouldn't work without the module installed, since you can't mock a command you don't have already), and the `Modules` entry in the attribute is for PSKoans to know how to sort / filter it.